pkg/kubernetes/deploy.js

```diff
@@ -76,6 +79,48 @@ define([
         $("#deploy-app-dialog").dialog("wait", promise);
     }
 
+    function deploy_nulecule() {
+        var promise = validate()
+            .fail(function(exs) {
+                console.log("validated failed");
+                $("#deploy-app-dialog").dialog("failure", exs);
+            })
+            .done(function(fields) {
+                promise = nulecule_client.create_tmp().done(function(tmp){
+                        console.log(tmp +" created");
+                        promise = nulecule_client.install(tmp, fields.nulecule_image).done(function() {
+                                /* code gets run when everything is created */
+                                //$('#deploy-app-dialog').modal('hide');
+                                console.log("install done");
+                            })
+                            .fail(function(ex, response) {
+                                console.log("install fail");
+                                var target;
+                                var msg;
+
+                                /* Display the error appropriately in the dialog */
+                                $("#deploy-app-dialog").dialog("failure", ex);
+                        });
+                        console.log("wait");
+                        console.log( promise.state());
+                        /* Display a spinner while this is happening */
```

Comment:
There seems to be a call to $("deploy-app-dialog").dialog("wait", promise) missing here...